packages/components/src/Combobox/Combobox.tsx

```diff
@@ -79,7 +79,6 @@ export function Combobox(props: ComboboxProps): JSX.Element {
           setSearchValue={setSearchValue}
           wrapperRef={wrapperRef}
           open={open}
-          setOpen={setOpen}
```

Comment:
this was not being used inside `ComboboxContent` at all. it was very likely a leftover from before we moved `setOpen` into the Context and moved things around.